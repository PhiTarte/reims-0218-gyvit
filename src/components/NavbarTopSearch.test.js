//TODO: create component search bar
