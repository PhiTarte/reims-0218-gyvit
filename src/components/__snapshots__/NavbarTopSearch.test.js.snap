// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="col-8 nav-item"
>
  <div
    className="form-group"
  >
    <span
      class="fa fa-search"
    />
    <input
      className="form-control"
      id="exampleSearch"
      name="search"
      placeholder="Search..."
      style={
        Object {
          "background-color": "#fbfcfd",
          "border": "none",
          "border-radius": "2px",
          "height": "46px",
          "marginLeft": "46px",
          "marginTop": "22px",
          "width": "1020px",
        }
      }
      type="search"
    />
  </div>
</li>
`;
